tests/refer-and-monitor-and-delius/reschedule-appointment.spec.ts

```diff
@@ -319,3 +321,31 @@ test('Reschedule Supplier Assessment Appointment to past date/time with attendan
         true
     )
 })
+
+test('Update Future Dated Supplier Assessment Appointment Location in Refer and Monitor and verify in Delius ', async ({
+    page,
+}) => {
+    const crn = await createOffender(page, { providerName: data.teams.referAndMonitorTestTeam.provider })
+    await createCommunityEvent(page, { crn, allocation: { team: data.teams.referAndMonitorTestTeam } })
+    await createRequirementForEvent(page, { crn, team: data.teams.referAndMonitorTestTeam })
+
+    // Create a Supplier Assessment Appointment in R&M with future date
+    const referralRef = await createAndAssignReferral(page, crn)
+    await createSupplierAssessmentAppointment(page, referralRef, addDays(new Date(), 2))
+
+    // Update Supplier Assessment Appointment Location in Refer and Monitor
+    await logoutRandM(page)
+    await loginRandMAsSupplier(page)
+    await updateSAAppointmentLocation(
+        page,
+        referralRef,
+        'In-person meeting - NPS offices',
+        'Alderdale: Progress House',
+        'Alderdale: Progress House'
+    )
+
+    // Verify that updated Supplier Assessment Appointment Location is available in Delius
+    await loginDelius(page)
+    await navigateToNSIContactDetails(page, crn)
+    await verifySAApptmntLocationInDelius(page, 'Appointment with CRS Staff (NS)', 'Workington - Progress House')
```

Comment:
Is it intentional that we're setting the office location to Alderdale in R&M, but verifying Workington in Delius?  Is it just mismatched reference data?